frontend/assets/locales/en/translation.json

```diff
@@ -1 +1 @@
-{"Start a group":"Start a group","New User":"New User","Profile":"Profile","Group":"Group","Pay Group":"Pay Group","EJS test":"EJS test","Sign Up":"Sign Up","Remember me":"Remember me","Login":"Login"}
+{"Start a group":"Start a group","New User":"New User","Profile":"Profile","Group":"Group","Pay Group":"Pay Group","EJS test":"EJS test","Sign Up":"Sign Up","Remember me":"Remember me","Login":"Login", "New notifications": "New {{num}} notifications"}
```

Comment:
Heh, you didn't need to update this file but this is OK. In the future don't update this file manually (it's something that will be auto-generated).